### ISSUE TITLE
201801119_Challenges_Solutions

### ISSUE THREAD

**Issue by khyatibhuva** [nisargptl/vehicle-recognition#64]
Write your challenges you faced in this entire project and how to solved them.

**Comment by khyatibhuva** [comment]
Q. As a group leader, making work everyone on same page was a challenge.
A. I always tried to find common time of all members to arrange meetings. Then I was to ask everyone to answer one question, so that all the views of the members were covered. I also used to keep in touch with almost everyday with at least one member of each team to know problems, solutions and progress. For distribution of work, I used to divide equally depending upon the workload of sprint and how much each member is able to handle and their preferences.

Q. As a member, In GUI, refining website was also a challenge.
A. Once we were done with basic functionalities, there were few features we wanted to implement like sticky, flex boxes, auto complete, adjustments of every division, alert boxes etc. That was a great amount of work put into refining and detailing because as we explored more, we wanted to use that into our website to make it user friendly and easy to use.

Q.  As a member and as a leader, managing both roles was also a challenge.
A. As a group leader, I had to be one step ahead of the team and as a member also I had to contribute equally without any comprise. To manage both, I devoted more time and efforts so that I can be part of every activity happening in project like their meetings, documentation, presentation, video etc. to make sure that everything is going well.
